### DISCUSSION TITLE
Help with VMW/AVGO transaction for TurboTax

### DISCUSSION THREAD

**Discussion by ronindran** [hickeng/financial#158] [General]
Hi all, I need help completing the VMW/AVGO transaction in TurboTax.  I've imported the forms into TurboTax from E*Trade and need help with completing it.  I have all the information that is required for the spreadsheet but I'm at a loss.  I've reached out to a few CPAs from the other thread but to no avail.  

Thanks in advance,
Ron

**Comment by hickeng** [comment]
> but I'm at a loss

Going to need more detail than that to help! Which step from [the instructions ](https://github.com/hickeng/financial/blob/main/usage.md) are you having issues with?
The only issue I'm aware of atm with the sheet is that the live AVGO price screws up some of the post-merger calcs because of the split - to correct that, update the price in Summary!M28 to be 10x the split price, ie. present it in pre-split terms.

Are you on the alumni slack - that's a little easier for live discussion as to where you have issues.

**Reply by ronindran** [reply]
Well, I'm going to need a whole lot of help.  I need someone do the spreadsheet for me... lol.  If I can get a CPA who would do this for me, that would be the ideal, but so far, my inquiries have gone nowhere.  It sounds complicated for a first time reader... lol.

**Reply by ronindran** [reply]
And, I absolutely don't mind paying for this work.

**Reply by ronindran** [reply]
I'm not on the alumni slack, unfortunately.  I should get on it.

**Reply by hickeng** [reply]
> It sounds complicated for a first time reader

It might be daunting, but so long as you've got access to your purchase confirmations for ESPP and RSUs it's not too bad - just an exercise in taking numbers from one document and copying them into the spreadsheet. Assuming you're US based that is. There's a different set of complexities that I'm not familiar with otherwise, but again the alumni slack has people who've gone through that.

When I say not that bad....
* 1 date and 1 share count per RSU lot
* 2 dates and 1 share count per ESPP lot
* 2 numbers for the overall transaction from eTrade transaction log (although you should add the validation amounts to help catch transcription errors)
* 1 checkbox for where any fraction came from (will be the oldest lot unless you ad adjusted the eTrade tax strategy)

See https://github.com/hickeng/financial/blob/main/usage.md#espp for an example of the data you need to move over. There's other optional data you can enter, but it's not needed for filing, only so all the financial is in one location for future reference if needed.

**Reply by ronindran** [reply]
I think I've input the numbers into the spreadsheet as best I can. I'm not sure if they are correct though... lol. I didn't hold any shares from ESPP through the merger; just RSUs.

The 1099-B I received from E*TRADE has only 3 items under 1099-B PROCEEDS FROM BROKER AND BARTER EXCHANGE TRANSACTIONS:

Long Term - Noncovered Securities
- BROADCOM INC - Cash for fraction (i.e., 0.8 shares of AVGO)

Noncovered Securities with Undetermined Holding Period:
- BROADCOM INC - Proceeds from the sale of AVGO shares received as part of the merger.
- VMW SHARES AZH26 - Cash portion of the transaction.

The next step is figure out how to enter these into TurboTax

**Reply by hickeng** [reply]
You should be able to ignore the 1099 from eTrade. They don't have the cost basis (which is correct - they shouldn't), but some people recieved 1099s that didn't have all their lots broken out or had 0s in other places. The supplement values were off for many people.

Use the purchase confirmations to populate the sheet for the lots you held over the merger.

[This section](https://github.com/hickeng/financial/blob/main/usage.md#entry-into-turbotax) illustrates how to enter the data into turbo tax. Just above that section is the general info about Form 8949 and has screenshots of the 3 places in the sheet you pull numbers from (2 for you because you don't have ESPP).

**Reply by hickeng** [reply]
Correction - ignore the 1099 for the purposes of completing the sheet. Import it into TT, then correct it as per those instructions I linked - that will result in the correct codes being added to the form for the updates.
